sources/iTermPreferencesBaseViewController.m

```diff
@@ -247,6 +264,14 @@ - (void)updateValueForInfo:(PreferenceInfo *)info {
             break;
         }
 
+        case kPreferenceInfoTypeUnsignedIntegerTextField: {
+            assert([info.control isKindOfClass:[NSTextField class]]);
+            NSTextField *field = (NSTextField *)info.control;
+            // FIXME: What is correct here? There is no unsignedIntegerValue in NSTextField.
+            field.doubleValue = [self uintForKey:info.key];
```

Comment:
You could do `field.stringValue = [NSString stringWithFormat:@"%lu", [self unsignedIntegerForKey:info.key]];`